docs/node.md

```diff
@@ -4,4 +4,141 @@ title: Setup full node
 sidebar_label: Setup full node
 ---
 
-TBD
\ No newline at end of file
+## Startup
+
+To start the node, run its executable:
+
+```bash
+radicle-registry-node
+```
+
+The node must store data on your hard drive.
+You can override the default location with a parameter:
+
+```bash
+radicle-registry-node --data-path <PATH>
+```
+
+### Connecting to the network
+
+The node connects by default to the FFNet network.
+It has a list of known nodes, which can be called to join the network for the first time.
+You can use your own list of bootstrap nodes with a startup parameter:
+
+```bash
+radicle-registry-node --bootnodes <ADDR>
+```
+
+## Mining Blocks
+
+You can support the network by mining blocks. Miners receive rewards from
+successfully mined blocks. To collect your rewards, you need an account.
+
+Generate an account for mining:
+
+```bash
+radicle-registry-cli account generate mining
+```
+
+This will print the SS58 address for your mining account.
+You can now run a mining node:
+
+```bash
+radicle-registry-node --miner <address>
+```
+
+If your node successfully mined a block and imported it will log “Imported own
+block”:
+```
+15:50:13.041 INFO radicle_registry_node::service  Imported own block #1322 (0xefd1…4445)
+```
+Depending on how lucky you are mining a block might take minutes or hours.
+
+When your node has mined a block, your mining account receives some block
+rewards. Check your balance with
+
+```bash
+radicle-registry-cli account balance <address>
+```
+
+At the moment, only single-threaded mining is supported. Follow [this
+issue](https://github.com/radicle-dev/radicle-registry/issues/298) for updates.
+
+## The node key
+
+Every node has a key pair, which is used to identify it on the P2P network.
+It's useful for building ties with other nodes.
+By default the key pair is generated on every startup, but you can set it manually to store and
+reuse it later.
+To do that, you must pass the private key file on the node startup:
+
+```bash
+radicle-registry-node --node-key-file <KEY_FILE>
+```
+
+If the file doesn't exist, it's going to be created and filled with a new random key.
+
+## Monitoring
+
+The node has multiple means of monitoring its state.
+
+### Logging
+
+The most basic monitoring is done with logging to the output.
+It's enabled by default, but you can change the verbosity on the node startup with an environment
+variable `RUST_LOG`.
+It can be set to one of these values, from least to most verbose:
+
+```
+OFF
+ERROR
+WARN
+INFO
+DEBUG
+TRACE
+```
+
+Alternatively it can be set to a number from 0 to 5.
+The default is `INFO` or 3 with some exceptions for logs useful only for developers.
+
+An example of limiting logging to `ERROR`:
+
+```bash
+RUST_LOG=ERROR radicle-registry-node
+```
+
+### Polkadot telemetry
```

Comment:
I’d like to remove this section and not support the polkadot telemetry

Reply:
Should we remove this option from node's CLI then?

Reply:
Yes

Reply:
Fixed